app/Commands/EditEnvironmentVariables.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace App\Commands;
+
+use App\Helpers\Helpers;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+class EditEnvironmentVariables extends Command
+{
+    /**
+     * The signature of the command.
+     *
+     * @var string
+     */
+    protected $signature = 'aws:edit-environment-variables';
```

Comment:
In keeping relatively consistent with the existing commands, would be good to rename this to

```aws:ecs:manage-env```

Thoughts?

Reply:
Done